### ISSUE TITLE
Fix duplicated procedure documentation for new Zenodo archives

### ISSUE THREAD

**Issue by aesharpe** [catalyst-cooperative/pudl-archiver#15]
Right now there are two descriptions of how to add new archives to Zenodo. One in the [`pudl-zenodo-storage` `README`,](https://github.com/catalyst-cooperative/pudl-zenodo-storage) and one in the PUDL ["Development/Working With the Datastore"](https://catalystcoop-pudl.readthedocs.io/en/dev/dev/datastore.html?highlight=uuid#prepare-zenodo-store) section of the read-the-docs. This information should only live in one place, the `pudl-zenodo-storage` `README`. 

This issue is dedicated to: 
1. Making sure that the `pudl-zenodo-storage` `README` is up to par
2. Removing all the detail from the PUDL docs description and simply referencing the `pudl-zenodo-storage` `README`.

**Comment by jdangerx** [comment]
Looks like `pudl-zenodo-storage` is now deprecated in favor of `pudl-archiver`; the docs in `pudl/docs/dev/datastore.rst` appear to already be out of date. Should we just point towards `pudl-archiver` in those docs now?

**Comment by zaneselvans** [comment]
Yes, anything that previously talked about the `pudl-archiver` or `pudl-zenodo-storage` tools / repos should now be pointing at this repo, as they were merged / refactored to work more closely together (creating this repo).

**Comment by jdangerx** [comment]
I'm taking it upon myself to clean up the `pudl` docs then.
